configure.acr

```diff
@@ -55,6 +55,7 @@ IFAND HAVE_LIB_Z USE_ZIP {
 ARG_WITHOUT WITH_GPL gpl do not build GPL code (grub, cxx, ... ) ;
 
 CHKDECL sys/personality.h ADDR_NO_RANDOMIZE
+CHKFNC c arc4random_buf
```

Comment:
rerun autoget.sh with latest acr from git to get the updated configure script

also, we should support the same for meson